Src/EmbeddedScripts.Python.PythonNet/PythonMultithreadingScope.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Python.Runtime;
+
+namespace EmbeddedScripts.Python.PythonNet
+{
+    public class PythonMultithreadingScope : IDisposable
+    {
+        private readonly IntPtr _threadState;
+
+        public PythonMultithreadingScope()
+        {
+            PythonEngine.Initialize();
```

Comment:
Looks like it should be global configuration (not per-CodeRunner instances) as it's using static (global) initializers.